css/index.css

```diff
@@ -1,19 +1,75 @@
 /* Header */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+/* Menu */
+.menu {
+  background: #dedede;
+  padding: 10px 20px;
+  background: #fff;
+}
 
+.navbar {
+  display: flex;
+}
 
-/* Menu */
+.navbar .navbar-branch {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 20%;
+  font-size: 18px;
+}
 
+.navbar .navbar-branch a {
+  text-decoration: none;
+  color: #5e5e5e;
+}
 
-/* Cover */
+.navbar .navbar-menu {
+  display: flex;
+  padding: 10px 20px;
+  width: 75%;
+  justify-content: flex-end;
+}
 
+.navbar .navbar-menu ul {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
 
-/* Main */
+.navbar .navbar-menu ul li {
+  list-style-type: none;
+  padding: 10px 20px;
+  text-align: center;
+  margin: 0 5px;
+  border-radius: 5%;
+  text-transform: uppercase;
+}
 
+.navbar .navbar-menu ul li:first-child {
```

Comment:
This is not correct, should change color based on active location(url)
Or do not implement at this time